common.sh

```diff
@@ -160,7 +160,10 @@ export MIRROR_LOG_FILE=${REGISTRY_DIR}/${CLUSTER_NAME}-image_mirror-${OPENSHIFT_
 # Switch Container Images to upstream, Installer defaults these to the openshift version
 if [ "${UPSTREAM_IRONIC:-false}" != "false" ] ; then
     export IRONIC_LOCAL_IMAGE=${IRONIC_LOCAL_IMAGE:-"quay.io/metal3-io/ironic:master"}
-    export IRONIC_INSPECTOR_LOCAL_IMAGE=${IRONIC_INSPECTOR_LOCAL_IMAGE:-"quay.io/metal3-io/ironic-inspector:master"}
+# Starting from Openshift 4.9 the ironic-inspector container is not used anymore
+    if [ "$OPENSHIFT_VERSION" <= "4.8" ]; then
```

Comment:
This isn't a valid test - I think you need something like `if [[ "$OPENSHIFT_VERSION" < "4.9" ]]` - that will still only do a string comparison, but that's probably good enough in this case

Reply:
yeah, I know this, I don't know what I was thinking :/